demos/tutorial/alphabets/assignment_1_solution.cpp

```diff
@@ -16,7 +16,7 @@ void showAllLettersOfMyAlphabet(TAlphabet const &)
     // We iterate over all characters of the alphabet
     // and output them
     for (TSize i = 0; i < alphSize; ++i)
-        std::cout << i << ',' << TAlphabet(i) << "  ";
+        std::cout << static_cast<unsigned>(i) << ',' << TAlphabet(i) << "  ";
```

Comment:
Because the type is unsigned char which is transformed to an unprintable char upon cout. The .stdout is labeled as binary by github for exactly this reason (it had not printable characters before).